karapuzio/assignment2/q2_common_ancestors/BinaryTree.java

```diff
@@ -0,0 +1,143 @@
+package edu.codeU.assignment2.CommonAncestor;
+
+import edu.codeU.assignment2.BinarySearchElementExeption;
+import edu.codeU.assignment2.BinaryTreeInitializationExceprion;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class BinaryTree<T> {
```

Comment:
It seems like you tried using generics here. Can you try and actually use them in your BinaryTree class?